### DISCUSSION TITLE
Intellij and Copilot - windows user without admin permissions

### DISCUSSION THREAD

**Discussion by drewoko** [community/community#34671] [Copilot Conversations]
Intellij : IntelliJ IDEA 2022.2.1 (Build #IU-222.3739.54)
Copilot: 1.1.32.1956
Windows: 19044.2006

Intellij throws an error message.

**Comment by bmuskalla** [comment]
Would you be so kind to share the error message?

**Reply by drewoko** [reply]
![image](https://user-images.githubusercontent.com/788188/193503767-fba6052a-3700-47fb-b6e2-52e662e567be.png)

**Reply by bmuskalla** [reply]
Do you see anything in the Intellij logs related to Copilot?
